Classes/Controller/ModuleController.php

```diff
@@ -281,7 +222,54 @@ public function clipboardAction()
      */
     public function linksAction()
     {
+        $backendUriBuilder = GeneralUtility::makeInstance(\TYPO3\CMS\Backend\Routing\UriBuilder::class);
```

Comment:
We should `use` this. `as BackendUriBuilder` if necessary.

Reply:
what do you mean? Is there a class called "BackendUriBuilder"?

Reply:
Yes, if you name it accordingly:

```
use TYPO3\CMS\Backend\Routing\UriBuilder as BackendUriBuilder;
```

This way you avoid conflicts and confusion with the Extbase `UriBuilder` (which you could also rename to `ExtbaseUriBuilder` instead but that'd be rather unusual.)

Reply:
Ok done